### PR TITLE
CB-9237 Add cdvfile:// support to media plugin on windows platform

### PR DESCRIPTION
**Pull request by daserge**
Mentioned cdvfile support in the documentation

[JIRA issue](https://issues.apache.org/jira/browse/CB-9237)

### CODE REVIEW

File: src/windows/MediaProxy.js [hunk 1]
```diff
@@ -35,6 +46,8 @@ module.exports = {
         var src = args[1];
         var thisM = Media.get(id);
 
+        src = cdvfileToNative(src);
```
Comment (nikhilkh) [inline src/windows/MediaProxy.js]: Should `src` for `startRecordingAudio` also be updated?

Comment (daserge) [inline src/windows/MediaProxy.js]: Yes, thanks!
Although I think of moving this commit to [CB-9238 Media plugin cannot record audio on windows](https://github.com/apache/cordova-plugin-media/pull/58) PR as it requires paths handling introduced there.

**Comment by dblotsky** [comment]
Ping. Please rebase atop latest master changes, and verify that it still works. Then we can merge.

**Comment by daserge** [comment]
@dblotsky, rebased and tested

**Comment by sgrebnov** [comment]
reviewed, lgtm :+1:

**Pull request closed without merging**